executor/executor_write.go

```diff
@@ -637,6 +641,7 @@ func (e *InsertValues) fillRowData(cols []*column.Col, vals []types.Datum) ([]ty
 }
 
 func (e *InsertValues) initDefaultValues(row []types.Datum, marked map[int]struct{}) error {
+	var pkHandleCol *column.Col
```

Comment:
I think rename it to `rewriteValueCol` would be better. check `pkHandleCol`is set indicates there is pk handle column, but it is set only when it has allocated a record ID.